packages/@n8n/task-runner/src/js-task-runner/js-task-runner.ts

```diff
@@ -335,4 +321,14 @@ export class JsTaskRunner extends TaskRunner {
 
 		return new ExecutionError({ message: JSON.stringify(error) });
 	}
+
+	private reconstructTaskData(response: DataRequestResponse): JsTaskData {
+		return {
+			...response,
```

Comment:
Should we mutate instead of copying? Or doesn't make enough of a difference?

Reply:
Thought about that but opted for copying to make it easier to reason about and make tests easier to write